src/test/java/uk/gov/digital/ho/proving/income/audit/AuditResultConsolidatorIT.java

```diff
@@ -132,19 +138,146 @@ public void byCorrelationId_notFoundResponseOnly_allDetailsFilled() {
         assertThat(results.get(0).resultType()).isEqualTo(NOTFOUND);
     }
 
-    private List<AuditRecord> loadJson(Resource... resourceFiles) {
-        return Arrays.stream(resourceFiles)
-            .map(resource -> FileUtils.loadJsonResource(resource))
-            .map(this::readAuditRecord)
-            .collect(Collectors.toList());
+    @Test
+    public void byCorrelationId_multipleRequestResponses_allDetailsFilled() {
+        AuditRecord auditRecord1Request = fileUtils.buildRequestRecord("3743b803-bd87-4518-8cae-d5b3e0566396", "2019-02-25 12:01:02.003", "PJ151008C");
+        AuditRecord auditRecord1Response = fileUtils.buildResponseNotFoundRecord("3743b803-bd87-4518-8cae-d5b3e0566396", "2019-02-25 12:01:02.003");
+        AuditRecord auditRecord2Request = fileUtils.buildRequestRecord("5e6d002f-fd09-4347-a7da-2cd23346da49", "2019-02-26 12:01:02.003", "PP151005D");
+        AuditRecord auditRecord2Response = fileUtils.buildResponseRecord("5e6d002f-fd09-4347-a7da-2cd23346da49", "2019-02-26 12:01:02.003", "PP151005D", "true");
+        List<AuditRecord> records = Arrays.asList(auditRecord1Request, auditRecord1Response, auditRecord2Request, auditRecord2Response);
+
+        List<AuditResult> results = auditResultConsolidator.auditResultsByCorrelationId(records);
+
+        AuditResult expected1 = getExpectedAuditResult1();
+        AuditResult expected2 = getExpectedAuditResult2();
+
+        assertThat(results.size()).isEqualTo(2);
```

Comment:
containsExactlyInAnyOrder will fail if there's not 2 elements, so this line can go.